app/constants/colorsRocketChat.js

```diff
@@ -0,0 +1,77 @@
+import { isIOS } from '../utils/deviceInfo';
```

Comment:
Can you make a pithy comment in this file about how to use it when you want to customize your own color scheme?

something like:
// Rename this file and substitute your own color scheme.  Then be sure to replace the import of this file in colors.js with your new file.

Reply:
Yeah I will add that

Reply:
>I think they would have to rename the file something like colorsMyOrg.js, and then import that into colors.js, right?

Looks like I miss worded it this is what I meant to say.